src/views/Swap/Swap.vue

```diff
@@ -626,15 +628,19 @@ export default {
         return 'Please increase amount. It is below minimum.'
       }
 
+      if (this.selectedQuote?.maxFeePercentage && (BN(this.selectedQuote.toAmount).times(this.selectedQuote?.maxFeePercentage)).lt(this.selectedQuote.toSwapFees)) {
```

Comment:
I don't think we need this, there is already a mechanism during swap review page that tells you if the swap you are making makes sense considering the fee
e.g. : 
![image](https://user-images.githubusercontent.com/11529637/141779834-5f8e5ce6-1ad3-4d15-90c7-5c4acca80f1b.png)

Reply:
We had a talk with Simon and Alok and we decided to do it like this. 
If the fee is more than 15% of the received amount to show a message and block the swap.

Reply:
I understand but how is that different from the mechanism in the screenshot?

I mean we shouldn't have 2 different ways of doing the same thing. If we want to block swap from happening when fee is too high, we should define a specific threshold and enforce it for all swaps the same way